src/modules/ProjectsSafePeople/ProjectsSafePeople.tsx

```diff
@@ -94,7 +94,7 @@ export default function ProjectsSafePeople({
     <T extends ProjectUser>(info: CellContext<T, unknown>) => {
       return (
         <Box sx={{ display: "flex" }}>
-          {renderUserNameCell(info.getValue() as User, userPath)}
+          {renderUserNameCell(info.getValue() as User, userPath, {projectId: project.id})}
```

Comment:
Fixes an unrelated bug where the project id was not being injected into the path.